src/compas_rhino/conduits/base.py

```diff
@@ -34,30 +49,56 @@ def enabled(self):
             self.disable()
 
     def CalculateBoundingBox(self, e):
+        """Calculate the model extents that should be included in the visualization.
+
+        Parameters
+        ----------
+        e : Rhino.DisplayCalculateBoundingBoxEventArgs
+
+        Returns
+        -------
+        None
+
+        """
         bbox = Rhino.Geometry.BoundingBox(-1000, -1000, -1000, 1000, 1000, 1000)
         e.IncludeBoundingBox(bbox)
 
     def enable(self):
-        """Enable the conduit."""
+        """Enable the conduit.
+
+        Returns
+        -------
+        None
+
+        """
         self.Enabled = True
 
     def disable(self):
-        """Disable the conduit."""
+        """Disable the conduit.
+
+        Returns
+        -------
+        None
+
+        """
         self.Enabled = False
 
-    def redraw(self, k=0, pause=None):
+    def redraw(self, k=0, pause=0.0):
```

Comment:
if the default changed to `0.0`, I would also change the line 106 below, to be a numeric check: `if pause > 0.0:` (which also hints at the fact we should validate pause not being negative)